libs/community/langchain_community/llms/bedrock.py

```diff
@@ -191,6 +191,11 @@ class BedrockBase(BaseModel, ABC):
         "cohere": "stop_sequences",
     }
 
+    guardrails: Optional[Mapping[str, str]] = {
```

Comment:
I would add a string doc on what this property represents (as it's not trivial)

Reply:
Thank you for pointing that out, @gilinachum. Recognizing the complexity of the property, I've updated it with a detailed docstring for better clarity on its purpose and usage.